src/Test.UnitTests.BinaryParsers/ELF/ELFBinaryTests.cs

```diff
@@ -88,5 +88,45 @@ public void ValidateDwarfV5_WithO2_Split_DebugFileMissing()
             binary.GetDwarfVersion().Should().Be(5);
             binary.GetLanguage().Should().Be(DwarfLanguage.Unknown); //missing dwo file should not cause exception
         }
+
+        [Fact]
+        public void ValidateDwarfV5_WithO2_No_Stack_Clash_Protection()

```

Comment:
>ValidateDwarfV5_WithO2_No_Stack_Clash_Protection [](http://example.com/codeflow?start=20&length=48)

you don't need to add this. The tests above were used to test the ELFBinary, and since we know it works, this isn't needed since you are already testing in the functionaltests.

Reply:
sounds good, removed.